app/src/main/java/ar/com/wolox/android/cookbook/googlelogin/GoogleLoginRecipeView.kt

```diff
@@ -8,5 +8,5 @@ interface GoogleLoginRecipeView {
 
     fun showNoUser()
 
-    fun showGoogleLoginError()
+    fun showGoogleLoginError(errorCode: Int?)
```

Comment:
We never pass the `errorCode` to the `View`. That's an implementation detail that has to do with the `Presenter` doing a network operation that may error out, but it's not a detail the `View` should be concerned with. We should have a method per type of error and the `Presenter` should decide internally which to call in each case.